package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lint",
+  "private": true,
+  "author": "Hassan Ali <helfi92@gmail.com>",
+  "engines": {
+    "node": "^8.10 || >=10",
+    "npm": ">=5.4.0",
+    "yarn": ">=1.10.1"
+  },
+  "scripts": {
+    "lint": "eslint --cache --format codeframe --ext js packages/*",
+    "test": "yarn lint",
+    "verdaccio:release": "lerna publish --registry http://localhost:4873 --yes --no-verify-access --no-git-tag-version --no-push --exact"
```

Comment:
I wonder if this is the recommended command to run when releasing scoped packages on verdaccio. That's what I've been doing.